hummingbot/client/config/config_helpers.py

```diff
@@ -177,20 +177,25 @@ def get_erc20_token_addresses() -> Dict[str, List]:
     for token in decoded_resp["tokens"]:
         token_list[token["symbol"]] = [token["address"], token["decimals"]]
 
-    with open(address_file_path) as f:
-        try:
+    try:
+        with open(address_file_path) as f:
             overrides: Dict[str, str] = json.load(f)
             for token, address in overrides.items():
                 override_token = token_list.get(token, [address, 18])
                 token_list[token] = [address, override_token[1]]
                 override_list.append(token)
 
             if len(override_list) > 0:
-                print(f"Applied address overrides from /conf/erc20_tokens.json to symbols {override_list}.")
+                print(f"Applied address overrides from {TOKEN_ADDRESSES_FILE_PATH} to symbols {override_list}.")
```

Comment:
better to log the message if this message is meant to be logged (into the log file).

Reply:
Initializing the logger in this function isn't really necessary. The config helper file contains no class but consists of helper functions which are used throughout the bot. Besides the printed info are not really important, any error that occurs anywhere in this function or other functions in this file will certainly be logged(which I believe is most important for any debugging).